Cards/CardContainer.gd

```diff
@@ -321,4 +325,28 @@ func _update_card_positions() -> void:
 				else:
 					card_x += scaled_hover_offset
 		
-		movement_component.state_properties.desired_position.x = card_x
+		# scale card's x value to a range of [-1, 1] from range of [0, viewport width]
+		var viewport_width: float = get_viewport_rect().size.x
+		var card_index_scaled: float = Helpers.convert_from_range(card.global_position.x, 0.0, viewport_width, -1.0, 1.0)
```

Comment:
this says that the card_index is being scaled to -1,1 but i feel like the viewport width is being scaled to -1,1 (which doesn't feel like a card index, ie the number of the card in the hand)

Reply:
Yeah, this is an old naming. I believe I updated this naming in the latest version